src/lib/pipeline_configuration.py

```diff
@@ -65,7 +65,9 @@ def __init__(self, raw_field, coded_field, coda_filename, cleaner=None, code_sch
 
 
 class PipelineConfiguration(object):
-    DEV_MODE = True
+    DEV_MODE = False
+
+    SUBSAMPLING_THRESHOLD = 4
```

Comment:
Comment that this is a fraction of hex

E.g.
`SUBSAMPLING_THRESHOLD = 4 # /16 subsample of data`

Reply:
Updated thanks